### DISCUSSION TITLE
Which Feature do you need the most?

### DISCUSSION THREAD

**Discussion by maxnowack** [maxnowack/signaldb#398] [Polls]
I already have a few ideas for future features of SignalDB. Let me know which one you find most interesting. If you have other feature requests, let me know in the comments or open a new issue.

**Comment by maxnowack** [comment]
Data replication is now released 🎉

**Comment by morgs32** [comment]
I think a little prisma studio-style GUI in devtools would be pretty nifty?

**Reply by maxnowack** [reply]
Yes, absolutely! [I already had something like this in my mind](https://github.com/maxnowack/signaldb/issues/396#issuecomment-2016706237) 🙂

**Reply by maxnowack** [reply]
I've released the devtools a while ago: https://signaldb.js.org/devtools/

**Comment by lorof** [comment]
How about implementing leader election for the sync manager?

**Comment by jswhisperer** [comment]
Would love the opt-in ability to use a schema like prismaORM or just a Zod Collection

**Reply by maxnowack** [reply]
Here is an example how SignalDB could be used together with zod: https://signaldb.js.org/schema-validation/

**Reply by jswhisperer** [reply]
Ah perfect thank you kindly

**Comment by maxfriedmann** [comment]
I'd love to see fulltext/fuzzy search integrated into signaldb

**Reply by jswhisperer** [reply]
Wouldn't fuse js cover this pretty well? Just curious

**Reply by maxfriedmann** [reply]
I thought so too, but fuse.js got quite slow with a lot of data and it does not offer persistent indexes ootb as far as I know. Do you have any experience there? I started a separate discussion here: https://github.com/maxnowack/signaldb/discussions/1678
